auth/cookie_manager.go

```diff
@@ -194,3 +199,10 @@ func (c CookieManager) DeleteCookies(ctx context.Context, writer http.ResponseWr
 	http.SetCookie(writer, getLogoutAccessCookie())
 	http.SetCookie(writer, getLogoutRefreshCookie())
 }
+
+func (c CookieManager) getCookieDomain(request *http.Request) string {
+	if !c.coverSubDomains {
+		return ""
+	}
+	return fmt.Sprintf(".%s", request.URL.Hostname())
```

Comment:
nice! I guess this works because this is an HTTP request not a GRPC request...

Have you tried it behind an ingress? mind doing that?

Reply:
Tested this both with grpc endpoints through flytectl and through flyteconsole and no issues were discovered